sdk/eventhub/azure-eventhubs/tests/asynctests/test_negative_async.py

```diff
@@ -49,6 +51,7 @@
     sender = client.create_producer()
```

Comment:
Isn't the sender a context manager? If so, should this be `async with client.create_producer() as sender:`?

Reply:
yup, it can be done as you described.
We also support manually closing the producer/consumer, either way would work.